src/axom/sidre/core/Group.cpp

```diff
@@ -1208,6 +1208,50 @@ bool Group::createNativeLayout(Node& n, const Attribute* attr) const
   return hasSavedViews;
 }
 
+/*
+ *************************************************************************
+ *
+ * Copy Group layout to the given Conduit node, including only the metadata
+ * for Views and not the actual data the Views hold.
+ *
+ *************************************************************************
+ */
+
+void Group::createNoDataLayout(Node& n, const Attribute* attr) const
+{
+  n.set(DataType::object());
+
+  // Dump the group's views
+  IndexType vidx = getFirstValidViewIndex();
+  while(indexIsValid(vidx))
+  {
+    const View* view = getView(vidx);
+
+    // Check that the view's name is not also a child group name
+    SLIC_CHECK_MSG(m_is_list || !hasChildGroup(view->getName()),
+                   SIDRE_GROUP_LOG_PREPEND
+                     << "'" << view->getName()
+                     << "' is the name of both a group and a view.");
+
+    if(attr == nullptr || view->hasAttributeValue(attr))
+    {
+      conduit::Node& child_node = m_is_list ? n.append() : n[view->getName()];
+      view->copyMetadataToNode(child_node);
+    }
+    vidx = getNextValidViewIndex(vidx);
+  }
+
+  // Recursively dump the child groups
+  IndexType gidx = getFirstValidGroupIndex();
+  while(indexIsValid(gidx))
+  {
+    const Group* group = getGroup(gidx);
+    conduit::Node& child_node = m_is_list ? n.append() : n[group->getName()];
+    group->createNoDataLayout(child_node, attr);
+    gidx = getNextValidGroupIndex(gidx);
+  }
```

Comment:
Suggestion: This might be cleaner with the new iterator interface for looping through views and groups